CITATION.cff

```diff
@@ -0,0 +1,18 @@
+# file: CITATION.cff
+cff-version: 1.2.0
+message: "If you use this software, please cite it as below."
+authors:
+- family-names: "Yadan"
+  given-names: "Omry"
+  orcid: "https://orcid.org/0000-0002-1871-7216"
+- family-names: "Sommer-Simpson"
+  given-names: "Jasha"
+  orcid: "https://orcid.org/0000-0002-1397-6454"
+- family-names: "Delalleau"
+  given-names: "Olivier"
+  orcid: "https://orcid.org/0000-0002-0610-7226"
+title: "omegaconf"
+#version: 2.1.1
+#doi: 10.5281/zenodo.1234
+#date-released: 2021-08-17
```

Comment:
@omry I left it in case you would like to add those fields in future. I left another comment in the related issue about creating a zenodo doi. Could you please take a look at it and let me know if you would like to add zenodo doi?

About the other two fields, it’s okay to drop them.